src/components/loader/loader-square.style.ts

```diff
@@ -8,6 +8,10 @@ export interface StyledLoaderSquareProps {
   isInsideButton?: boolean;
   /** Applies slate color. Available only when isInsideButton is true. */
   isActive?: boolean;
+  /** Toggle between the default variant and gradient variant */
+  variant?: string;
```

Comment:
**suggestion**: because you are conditionally setting the background colour in `loader.component.tsx` depending on `variant` , we could remove `variant` as a prop for this styled component:

```suggestion
```

Reply:
We extend `StyledLoaderSquareProps` in the component file, so I just left it in there, but i'm happy to move it back into the component file, however I thought it wouldn't make much difference

Reply:
Agree with @Parsium let's move that as it's irrelevant to the styled component now